docs/DeveloperGuide.adoc

```diff
@@ -279,13 +279,13 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
+* is a student of National University of Singapore (NUS)
```

Comment:
Should we be more specific to say SOC students

Reply:
I think our features are general enough to apply to all students?

Reply:
can, we can go ahead with general students of nus